demo/Makefile

```diff
@@ -25,11 +25,12 @@
 TAG=$(shell perl -n -e 'if (/docker-workflow:(.+)/) {print $$1}' plugins.txt)
 IMAGE=jenkinsci/docker-workflow-demo
 
-build-proxy:
-	docker build -t nginx:docker-workflow-demo -f Dockerfile-proxy .
+build-registry:
+	./gen-security-data.sh certs
```

Comment:
Logically would belong in `Dockerfile-registry`, but the `registry` image does not seem to contain an `openssl` binary.

Alternately, could check in a static `*.crt` + `*.key`.